presentation/src/main/java/com/sopt/presentation/calendar/CalendarRoute.kt

```diff
@@ -117,30 +150,47 @@ fun CalendarRoute(
             },
             content = {
                 NavHost(navController, startDestination = SCHEDULE_LIST) {
-                    composable(SCHEDULE_LIST) { backStackEntry ->
-                        ScheduleListScreen(
-                            data = calendarViewModel.getSelectedScheduleEntity(),
-                            onItemClick = { schedule ->
-                                backStackEntry.savedStateHandle[SCHEDULE] =
-                                    schedule.id // 바꿔야 함
-                                navController.navigate(SCHEDULE_DETAIL)
-                            },
-                            onCreateAppointmentBtnClick = {
-                                calendarViewModel.navigateToAppointmentCreate()
-                                calendarViewModel.showBottomSheet(false)
-                            }
-                        )
+                    composable(SCHEDULE_LIST) {
+                        clickDate?.let { date ->
+                            ScheduleListScreen(
+                                data = ScheduleEntity(
+                                    groupId = calendarViewModel.selectedGroupId.value ?: -1,
+                                    date = date,
+                                    scheduleList = calendarViewModel.selectedDayAppointments.value.map {
+                                        CalendarAppointmentEntity(
+                                            id = it.id,
+                                            name = it.name,
+                                            category = it.category,
+                                            startTime = it.startTime,
+                                            endTime = it.endTime,
+                                            duration = it.duration,
+                                            date = it.date
+                                        )
+                                    }
+                                ),
+                                onItemClick = { id ->
+                                    clickAppointmentId = id
+                                    calendarViewModel.getConfirmedDetail(clickAppointmentId)
+                                    navController.navigate(SCHEDULE_DETAIL)
```

Comment:
P1: 빈 화면 위에 다이얼로그 뜨는 것보다 그룹 아이템이 보이는 상태에서 뜨는게 나을 것같아서 이렇게 수정해주쇼!!
```suggestion
                  when (getConfirmedDetailState) {
                                        is UiState.Success -> {
                                            navController.navigate(SCHEDULE_DETAIL)
                                        }

                                        else -> Unit
                                    }
```

Reply:
반영했슴다